ui/admin/app/templates/scopes/scope/targets/target/index.hbs

```diff
@@ -30,36 +30,42 @@
           @cancel={{route-action 'cancel' @model}}
         />
       </bc.Body>
-
       <bc.Sidebar>
-        <div class='target-sidebar-aliases'>
-          <h3>{{t 'resources.alias.title_plural'}}</h3>
-          <p class='description'>
-            {{t 'resources.alias.form.sidebar.help'}}
-          </p>
-          {{#if @model.aliases}}
+        {{#unless @model.aliases}}
+          {{#if (can 'create model' this.globalScope collection='aliases')}}
+            <div class='target-sidebar-aliases'>
```

Comment:
Let me know if there's a better without repeating the div

Reply:
I don't need the width anymore as I added that for when I used dropdown icon. good catch! I'm going to leave the refactor work for later as I do need the margin